src/libpsl-native/CMakeLists.txt

```diff
@@ -3,7 +3,8 @@ project(PSL-NATIVE)
 
 # Can't use add_compile_options with 2.8.11
 set(CMAKE_BUILD_TYPE "Release")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Werror -fstack-protector-strong -fpie")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Werror -fstack-protector-strong -fpie -DFORTIFY_SOURCE=2")
```

Comment:
Based on the documentation, FORTIFY_SOURCE=2 isn't enabled unless you also use `-O2` optimizations

Reply:
Resolved.